mjsimulateui/Makefile

```diff
@@ -0,0 +1,13 @@
+# This Makefile assumes that you have GLFW libraries and headers installed on,
+# which is commonly available through your distro's package manager.
+# On Debian and Ubuntu, GLFW can be installed via `apt install libglfw3-dev`.
+
+COMMON=-O2 -I../include -L../lib -std=c++11 -pthread -Wl,-rpath,'$$ORIGIN'/../lib
+
+all:
+	$(CC)  -c -O2 -fPIC -I../include uitools.c
+	$(CXX) -c -O2 -fPIC -I../include mjsimulateui.cc
+	$(CXX) -shared -o libmjsimulateui.so mjsimulateui.o uitools.o
+	mv libmjsimulateui.so ../lib/libmjsimulateui.so
```

Comment:
Should I just keep the shared library in this folder for now? Or should I put it in this directory as if the library were part of a full MuJoCo build?

Reply:
Maybe I should make seperate `lib`, `include` directories in `mjsimualteui`?